bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/DebugModeHandler.java

```diff
@@ -445,7 +506,8 @@ public void run() {
             }
         }
 
-        throw new Exception("Unable to connet to JVM on host: " + host + " and port: " + port);
+        throw new Exception("Unable to automatically attach the debugger to JVM on host: " + host + " and port: " + port
```

Comment:
"...manual action is required".   

So what can/should the user do at this point?  I think they could either just use the Remote Java App debug config with the port we mentioned... or they could just ignore debug altogether and just use it as if you did "Start".    If that's basically it maybe we could say a bit more about the first option.

Reply:
Updated.